source/administration/devops-command-center.rst

```diff
@@ -64,48 +64,126 @@ Viewing and creating playbooks
+5. Add a few checklist items.
+6. Select **Save**.
+7. Switch to the main channel view.
+8. Create an incident by selecting the playbook you just created.
 
-Choose **Done** to save the changes.
+Private incidents are invisible to non-participants: they do not appear in the RHS when active, nor anywhere in the backstage. Private incidents cannot be made public once they've been started. If an incident was public and changed to private, it can't be accessed by non-participants thereafter, and the contents of the channel won't show up in a search query.
+
+Editing checklists
+~~~~~~~~~~~~~~~~~~~
+
+You can edit checklist items and their position in the checklist during an active incident as well as checklist items in playbooks not currently being used in active incidents.
+
+1. Select **!** in the channel header of an active incident.
+2. Choose **Edit**.
+  - To edit: Select the text field to edit details.
+  - To delete: Hover over an input box and select the **X** to remove the checklist item.
+  - To move: Drag the hamburger menu to rearrange the checklist items.
+3. Choose **Done** to save the changes.
 
 Ending incidents
 ~~~~~~~~~~~~~~~~
 
 There are two ways to end an incident:
 
-- Using slash command */incident end* from within the incident channel.
+- Using the slash command */incident end* from within the incident channel.
 - Using the **End Incident** button in the RHS panel while in the incident channel.
 
-The incident will become inactive, be removed from the list of active incidents, and the associated channel will be archived.
+The incident will become inactive, is removed from the list of active incidents, and the associated channel is archived.
 
 Administrator's Guide
 ^^^^^^^^^^^^^^^^^^^^^^
 
+Incidents and playbooks are associated with teams in Mattermost. Participants can be added to an incident by other incident members inviting them to the incident's channel. By default, all incidents are public (available to all members of a team). An incident can be made private during creation, or it can be made private after it has been created. 
+
 Permissions
 ~~~~~~~~~~~~~~~~~~~~~
 
-Incidents and playbooks are associated with teams in Mattermost. Participants can be added by inviting them to the incident's channel.
+System Admins and Team Admins can edit the visibility of, and access to, playbooks and incidents so that:
+
+- Users who are removed from an incident channel by a System Admin can no longer access the incident's playbook.
+- Team members who aren't System or Team Admins can't invite other team members to manage private playbooks.
+- Team members who aren't System or Team Admins can't be removed from an incident by other members of the team.
+- Team members who aren't System or Team Admins can't have access to playbooks removed by other members of the team.
+
+In addition, System and Team Admins can allow:
+
+- Public playbooks to be deleted by team members whether they're participants in an incident channel or not.
```

Comment:
I think any playbook member can delete a playbook. Also, with `public playbook` do you mean playbooks that create public incidents?